.github/workflows/ci.yml

```
@@ -312,6 +318,11 @@ jobs:
     needs: [unit-test, lint, script-test, deterministic-gate]
     if: ${{ github.event_name != 'pull_request' || !github.event.pull_request.head.repo.fork }}
     runs-on: ubuntu-latest
+    # All live runs share one bot credential and remote resource pool.
+    concurrency:
+      group: lark-cli-e2e-live-shared-bot
+      cancel-in-progress: false
+      queue: max
```

actionlint (1.7.12)

[error] unexpected key "queue" for "concurrency" section. expected one of "cancel-in-progress", "group"

(syntax-check)